core/backend/src/ECSqlStatement.ts

```diff
@@ -689,6 +689,13 @@ export interface ECSqlColumnInfo {
    */
   getPropertyName(): string;
 
+  /** Gets the name of the original property that the column data is from.
+   * > Other than getPropertyName(), this ignores aliases and allows getting the name
```

Comment:
Use links for referenced APIs: `[[getPropertyName]]`. Consider updating documentation for `getPropertyName` to explain how it is affected by aliases etc.